File: CRUD/C#/Aplicacion Web con Ajax/sistema/sistema/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace sistema.Controllers
{
    public class HomeController : Controller
    {
        Conexiones.Conexion conexion_now = new Conexiones.Conexion();

        public ActionResult Index()
        {
            if (Request.Cookies["uid"] != null)
            {
                if (conexion_now.valid_cookie(Request.Cookies["uid"].Value))
                {
                    return RedirectToAction("Index", "Administracion");
                }
                else
                {
                    return RedirectToAction("LogOn", "Login");
                }
            }
            else
            {
                return RedirectToAction("LogOn", "Login");
            }

        }
    }
}

File: CRUD/Java usando NetBeans/Aplicacion Web usando MVC Spring V2/SistemaWebSpring/src/main/java/sistema/controllers/HomeController.java
// Written by Ismael Heredia in the year 2017

package sistema.controllers;

import javax.servlet.http.HttpServletRequest;
import org.springframework.context.ApplicationContext;
import org.springframework.context.support.ClassPathXmlApplicationContext;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.servlet.ModelAndView;
import sistema.bo.UsuarioBO;
import sistema.functions.Funciones;

@Controller
public class HomeController {

    ApplicationContext context
            = new ClassPathXmlApplicationContext("Spring-Module.xml");

    UsuarioBO usuarioBO = (UsuarioBO) context.getBean("UsuarioBO");

    Funciones funciones = new Funciones();

    @RequestMapping("/")
    public ModelAndView HomeForm(HttpServletRequest request) {
        if (funciones.valid_cookie(request)) {
            return new ModelAndView("redirect:/administracion");
        } else {
            return new ModelAndView("redirect:/login");
        }
    }

}
